screens/authentication/signUp/SignUpScreen.jsx

```diff
@@ -59,9 +59,10 @@ const SignUpScreen = ({navigation}) => {
     },
   });
 
-  const changeBackgroundColor = () =>
-    isChecked ? theme.colors.primary_green : theme.colors.light_gray;
-
+  const changeBackgroundColor = useMemo(
+    () => (isChecked ? theme.colors.primary_green : theme.colors.light_gray),
+    [isChecked],
+  );
   return (
```

Comment:
there is a handleBlur function you can get from useFormik instead of using setFieldTouched use handleBlur 

onBlur={() => setFieldTouched('email')} change this part of the code